### PR TITLE
fix: 「依存関係の事前バンドル」ページの翻訳の修正

### PR DESCRIPTION
**Pull request by sapphi-red**
「依存関係の事前バンドル」ページの翻訳をいくつか修正しました

### CODE REVIEW

File: guide/dep-pre-bundling.md [hunk 1]
```diff
@@ -32,11 +32,11 @@ Pre-bundling dependencies:
 依存関係の事前バンドルは開発モードでのみ適用され、依存関係を ESM に変換するために `esbuild` を使用します。本番ビルドでは、代わりに `@rollup/plugin-commonjs` が使用されます。
 :::
 
-## 自動依存関係の検出
+## 依存関係の自動検出
```
Comment (sapphi-red) [inline guide/dep-pre-bundling.md:35]: 原文

> Automatic Dependency Discovery

File: guide/dep-pre-bundling.md [hunk 2]
```diff
@@ -32,11 +32,11 @@ Pre-bundling dependencies:
...
 
-既存のキャッシュが見つからない場合、Vite はソースコードをクロールし、依存関係のインポートを自動的に検出します。（すなわち、`node_modules` から解決されることを期待されている "bare imports"）を探し、事前バンドルのエントリポイントとして使用します。事前バンドルは `esbuild` で実行されるので、通常は非常に高速です。
+既存のキャッシュが見つからない場合、Vite はソースコードをクロールし、依存関係のインポート（すなわち、`node_modules` から解決されることを期待されている "bare imports"）を自動的に検出します。そして、検出されたインポートを事前バンドルのエントリポイントとして使用します。事前バンドルは `esbuild` で実行されるので、例のごとく非常に高速です。
```
Comment (sapphi-red) [inline guide/dep-pre-bundling.md:37]: 原文

> If an existing cache is not found, Vite will crawl your source code and automatically discover dependency imports (i.e. "bare imports" that expect to be resolved from node_modules) and use these found imports as entry points for the pre-bundle. The pre-bundling is performed with esbuild so it's typically very fast.

---

かっこのかかる位置を修正して文を調整しました。

File: guide/dep-pre-bundling.md [hunk 3]
```diff
@@ -32,11 +32,11 @@ Pre-bundling dependencies:
...
 
-サーバを起動したあと、キャッシュにない新しい依存関係のインポートに遭遇した場合は、Vite は、依存関係管理ツールによる再バンドリングプロセスを実行し、ページをリロードします。
+サーバを起動したあと、キャッシュにない新しい依存関係のインポートに遭遇した場合は、Vite は依存関係の事前バンドルを再実行し、ページをリロードします。
```
Comment (sapphi-red) [inline guide/dep-pre-bundling.md:39]: 原文

> After the server has already started, if a new dependency import is encountered that isn't already in the cache, Vite will re-run the dep bundling process and reload the page.

---

「re-run the dep bundling process」の箇所の修正です。

File: guide/dep-pre-bundling.md [hunk 4]
```diff
@@ -65,7 +65,7 @@ export default defineConfig({
 
 ## 挙動のカスタマイズ
 
-デフォルトの依存関係発見の経験則は、必ずしも望ましいとは限りません。リストから依存関係を明示的に含めたり除外したりする場合は、[`optimizeDeps` 設定オプション](/config/#依存関係の最適化オプション)を使用してください。
+デフォルトのヒューリスティックな方法による依存関係の発見によって、必ずしも望ましい結果が得られるとは限りません。リストから依存関係を明示的に含めたり除外したりする場合は、[`optimizeDeps` 設定オプション](/config/#依存関係の最適化オプション)を使用してください。
```
Comment (sapphi-red) [inline guide/dep-pre-bundling.md:68]: 1文目の原文

> The default dependency discovery heuristics may not always be desirable.

File: guide/dep-pre-bundling.md [hunk 5]
```diff
@@ -75,20 +75,20 @@ export default defineConfig({
 
 ### File System キャッシュ
 
-Vite は、`node_modules/.vite` に、バンドル済みの依存関係をキャッシュします。これにより、いくつかのソースに基づいて、バンドル前のステップを再実行する必要があるかどうか:
+Vite は、`node_modules/.vite` に、事前バンドル済みの依存関係をキャッシュします。いくつかのソースに基づいて、事前バンドルを再実行する必要があるかどうか決定します:
```
Comment (sapphi-red) [inline guide/dep-pre-bundling.md:78]: 原文

> Vite caches the pre-bundled dependencies in `node_modules/.vite`. It determines whether it needs to re-run the pre-bundling step based on a few sources:

File: guide/dep-pre-bundling.md [hunk 6]
```diff
@@ -75,20 +75,20 @@ export default defineConfig({
...
 
 - `package.json` の `dependencies` リスト。
 - パッケージマネージャのロックファイル　例： `package-lock.json`、`yarn.lock`、または `pnpm-lock.yaml`。
-- vite.config.js に関連するフィールドがあれば、それを入力してください。
+- もし存在すれば、vite.config.js の関連するフィールド。
```
Comment (sapphi-red) [inline guide/dep-pre-bundling.md:82]: 原文

> - Relevant fields in your `vite.config.js`, if present.

File: guide/dep-pre-bundling.md [hunk 7]
```diff
@@ -75,20 +75,20 @@ export default defineConfig({
...
 
-上記のいずれかが変更された場合のみ、バンドル前のステップを再実行する必要があります。
+上記のいずれかが変更された場合のみ、事前バンドルを再実行する必要があります。
```
Comment (sapphi-red) [inline guide/dep-pre-bundling.md:84]: 原文

> The pre-bundling step will only need to be re-run when one of the above has changed.

File: guide/dep-pre-bundling.md [hunk 8]
```diff
@@ -75,20 +75,20 @@ export default defineConfig({
...
 
-何らかの理由で Vite に外れた場合の再バンドルを強制したい場合は、開発サーバを `--force` コマンドラインオプションで起動するか、手動で `node_modules/.vite` のキャッシュディレクトリを削除します。
+何らかの理由で Vite に再バンドルを強制したい場合は、開発サーバを `--force` コマンドラインオプションで起動するか、手動で `node_modules/.vite` のキャッシュディレクトリを削除します。
```
Comment (sapphi-red) [inline guide/dep-pre-bundling.md:86]: 原文

> If for some reason you want to force Vite to re-bundle deps, you can either start the dev server with the `--force` command line option, or manually delete the `node_modules/.vite` cache directory.

---

「you want to force Vite to re-bundle deps」の箇所の修正です

File: guide/dep-pre-bundling.md [hunk 9]
```diff
@@ -75,20 +75,20 @@ export default defineConfig({
...
 
 ### ブラウザ キャッシュ
 
-解決された依存関係のリクエストは、開発中のページの再読み込みのパフォーマンスを向上させるために、HTTP ヘッダ `max-age = 31536000、immutable` で積極的にキャッシュされます。キャッシュされると、これらのリクエストは開発サーバに再びヒットすることはありません。異なるバージョンがインストールされている（パッケージマネージャのロックファイルに反映されている）場合は、追加されたバージョンクエリによって自動的に無効になります。 デバッグしたい場合ローカルで編集することで、依存関係を作成できます:
+解決された依存関係のリクエストは、開発中のページの再読み込みのパフォーマンスを向上させるために、HTTP ヘッダ `max-age = 31536000、immutable` で積極的にキャッシュされます。一度キャッシュされると、これらのリクエストは開発サーバに再び到達することはありません。異なるバージョンがインストールされた（パッケージマネージャのロックファイルに反映された）場合は、付与されているバージョンクエリによって自動的に無効になります。ローカルでの編集で依存関係をデバッグしたい場合は、以下のように行えます:
```
Comment (sapphi-red) [inline guide/dep-pre-bundling.md:90]: 原文

> Resolved dependency requests are strongly cached with HTTP headers `max-age=31536000,immutable` to improve page reload performance during dev. Once cached, these requests will never hit the dev server again. They are auto invalidated by the appended version query if a different version is installed (as reflected in your package manager lockfile). If you want to debug your dependencies by making local edits, you can:

**Review by jay-es — APPROVED** [review]
ありがとうございます！:pray:

**Pull request merged**